2.4/root/usr/bin/run-mongod

```diff
@@ -47,9 +47,7 @@ function cleanup() {
   fi
   echo "=> Shutting down MongoDB server ..."
   if [ -f "${MONGODB_PID_FILE}" ]; then
-    kill -2 $(cat ${MONGODB_PID_FILE})
-  else
-    pkill -2 mongod
+    mongod --shutdown $mongo_common_args
```

Comment:
In other parts of scripts, there is also only `mongod --shutdown`...

https://github.com/openshift/mongodb/blob/master/2.6/root/usr/bin/run-mongod#L94
https://github.com/openshift/mongodb/blob/master/2.6/root/usr/bin/run-mongod#L86